docs/boilerplate/app/utils/useAppTheme.tsx.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 10
+---
+
+# useAppTheme()
```

Comment:
```suggestion
# useAppTheme
```

nit, I just noticed it looks inconsistent in the sidebar:

<img width="700" alt="Screenshot 2024-08-06 at 4 06 41 PM" src="https://github.com/user-attachments/assets/45b61533-8fb0-454f-a2ed-e24c2bcc09f0">